scripts/main.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener('DOMContentLoaded', () => {
+  console.log('imported');
+  const m = document.getElementById('switch-menu');
```

Comment:
[OPTIONAL] Kindly give a good and meaningful name to your variable so that when you or other developers want to work on your code, it can be easily understood.